### DISCUSSION TITLE
pedal hardware

### DISCUSSION THREAD

**Discussion by laenzlinger** [mikeoliphant/neural-amp-modeler-lv2#80] [Ideas]
Hi 

I am working on some robust open source hardware / software platform form pedals for a while. It's now in a state where I would like to invest more into application. Your project looks very interesting. Are you maybe interested in a collaboration? 

For more details see https://github.com/pedalboard

Best Regards
  Christof

**Comment by mikeoliphant** [comment]
Definitely keep me posted of your progress.

You may also be interested in my [Stompbox ](https://github.com/mikeoliphant/stompbox) project, and in the work by [TreeFallSound](https://www.treefallsound.com).

**Comment by laenzlinger** [comment]
Thanks for your response. 

My goal is a free hardware alternative of some commercial solutions (e.g. MOD). My latest work was focusing on the hardware. It is now on a good level. I have just recently finished  an alternative for the HifiBerry DAC ACD pro card. Now I have a complete stack of /midi processor inputs and displays and a reliable audio processor based on CM4

On the CM4 i am running ELK Audio, which results in very low latency audio processing and would like to use your plugin to try out more advance use cases which require a bit more processing power.  You mentioned that you were able to run it on Pi 4, so it should be feasible.

I am currently producing prototypes of the hardware in my own lab.  If you are **really** interested in a collaboration, let me know - I would send a prototype to you. Please contact me on [Discord](https://discord.gg/ncyKyryHAc).
